docs/site/soap-calculator-tutorial-scaffolding.md

```diff
@@ -0,0 +1,71 @@
+---
+lang: en
+title: 'App scaffolding'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/soap-calculator-tutorial-scaffolding.html
+---
+
+### Create your app scaffolding
+
+Let's start by creating the initial application by running the following
+command:
+
+```sh
+lb4 app soap-calculator --enableRepository
+```
+
+**Note:** The option **--enableRepository** instructs the **CLI** to include a
```

Comment:
This reminds me that we need to include a description for the flag in `Application-generator.md`.

Reply:
yes, IMO also the other tutorials need to be updated and use this flag instead of adding manually the code.